theme/icons/image_placeholder.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 700 250"><g fill="none" fill-rule="evenodd"><rect width="700" height="250" fill="#FAFAFA" rx="4"/><text fill="#5F6F77" font-family="Arial,sans-serif" font-size="24"><tspan x="247.9" y="135">Uploading image…</tspan></text></g></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 700 250"><g fill="#FAFAFA" fill-rule="evenodd"><rect width="700" height="250" rx="4"/></g></svg>
```

Comment:
BTW, why do we need that SVG? It's displaying a gray area, that's what it does? Couldn't we use a CSS element instead?

Reply:
> SVG file is still used as an image source while the upload is in progress because:

> 1. image can't have an empty source
> 2. svg file sets the size of the image and this size is scalable (e.g. when switching image to side image)